spec/requests/api/v1/events_request_spec.rb

```diff
@@ -44,4 +44,59 @@
     expect(results[:attributes][:image_url]).to eq(@event_2.image_url)
     expect(results[:attributes][:video_url]).to eq(@event_2.video_url)
   end
+  it 'returns events filtered by event_type' do
```

Comment:
The tests for this are very thorough. I think adding extra events to ensure we were only getting back what we expected was a good choice.